plugins/package-managers/pub/src/funTest/assets/projects/external/dart-http-expected-output.yml

```diff
@@ -959,20 +959,20 @@ packages:
       value: ""
       algorithm: ""
   source_artifact:
-    url: "https://pub.dev/packages/args/versions/2.5.0.tar.gz"
+    url: "https://pub.dev/packages/args/versions/2.6.0.tar.gz"
     hash:
-      value: "7cf60b9f0cc88203c5a190b4cd62a99feea42759a7fa695010eb5de1c0b2252a"
+      value: "bf9f5caeea8d8fe6721a9c358dd8a5c1947b27f1cfaa18b39c301273594919e6"
       algorithm: "SHA-256"
   vcs:
     type: "Git"
-    url: "https://github.com/dart-lang/args.git"
+    url: "https://github.com/dart-lang/core.git"
     revision: ""
-    path: ""
+    path: "main/pkgs/args"
   vcs_processed:
     type: "Git"
-    url: "https://github.com/dart-lang/args.git"
+    url: "https://github.com/dart-lang/core.git"
     revision: ""
-    path: ""
+    path: "main/pkgs/args"
```

Comment:
This path does not exist in the repository, but `pkgs/args` does. I assume `main` refers to the branch.

Reply:
I'll have a look.

Reply:
The data is indeed wrong [upstream](https://github.com/dart-lang/core/blob/58762460f95fd6ed4b9f30138be22863b742aef4/pkgs/args/pubspec.yaml#L6). I've filed an [issue](https://github.com/dart-lang/core/issues/138).